make_sitemap.js

```diff
@@ -17,13 +17,20 @@ const getPaths = async ({ distPath, exclude, cwd }) => {
   return paths
 }
 
-const getUrlFromFile = ({ file, distPath, prettyURLs, trailingSlash }) => {
-  const url = file.startsWith(distPath) ? file.replace(distPath, '') : distPath
+const normalizeFile = ({ file, distPath }) => {
```

Comment:
We could potentially replace this function by `path.relative`

Reply:
I am not sure considering those are relative URLs, not file paths. Would `new URL(uri, base)` be more proper?
For example, we'd need to use `path.posix.relative()` so that forward slashes are used for Windows users. 
I am also wondering whether using some string manipulation logic is simple enough, or whether you think there are some cases the current logic might fail?

Reply:
I'm good with keep the current logic unless we have more issues opened